src/jace/util/util.py

```diff
@@ -0,0 +1,36 @@
+# JaCe - JAX Just-In-Time compilation using DaCe (Data Centric Parallel Programming)
```

Comment:
The name of the module is not descriptive enough. I would personally merge it together with the `traits` modules (whose name I also find misleading in a python context) and rename it `definitions` or something like that.

Reply:
I do not think so. However, I see where you are coming from, so I renamed the `util.py` to `misc.py`.